torchao/quantization/quant_primitives.py

```diff
@@ -151,12 +144,12 @@ def quantize_affine(
       output_dtype (torch.dtype): requested dtype (e.g. torch.uint8) for output Tensor
       quant_min (Optional[int]): minimum quantized value for output Tensor, if not specified, it will be derived from dtype
       quant_max (Optional[int]): maximum quantized value for output Tensor, if not specified, it will be derived from dtype
-      zero_point_domain (ZeroPointDomain): the domain that zero_point is in, should be eitehr integer or float
+      zero_point_domain (str): the domain that zero_point is in, should be eitehr "int" for "float"
```

Comment:
oh it's because custom op API doesn't support enum right now

Reply:
@zou3519 any plans to add support for this in the API?

Reply:
No, the set of types supported by operators is intentionally limited so that people don't have a difficult time working with it (for example, writing graph passes on FX graphs with operators). You can wrap the call to the operator in a python function that does support enums

Reply:
> You can wrap the call to the operator in a python function that does support enums

oh, I see, maybe that's what we should be doing here